Automattic/wp-calypso
# Stats: Converting Shares component

Related to #72134 
## Proposed Changes

* replace the current `Shares` component to use horizontal bars
* extend the horizontal bar component to display shortened numbers for values

## Testing Instructions



* open the live branch
* apply `?flags=stats/insights-page-grid` to the URL
* verify that `Shares` module on the `Insights` page has horizontal bars
* verify that without the feature flag the current version of the component loads without problems

| Before | After |
|---|---|
| ![SCR-20230209-nrn](https://user-images.githubusercontent.com/112354940/217715378-2c1c5e79-834f-4340-8cb0-bea89b72467d.png) | ![SCR-20230209-nmq](https://user-images.githubusercontent.com/112354940/217715403-09bfa616-42d6-4d1b-a91c-75bf174edf3d.png) |

## Pre-merge Checklist
